src/appengine/handlers/cron/fuzz_strategy_selection.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Fuzzing strategy selection cron job.
+
+	Runs multi-armed bandit experiments for fuzzing strategy selection.
+	In particular, this is a Boltzman Exploration (softmax) implementaion
+  of multi-armed bandit experiments. Queries from bigquery to update
+  multi-armed bandit probability values based on the new edges for various
+  combined strategies. In the upload_bandit_weights function, we can change
+  metric to be for edges, crash, features, or units. Currently based on new
+  edges."""
+
+from datastore import data_types
+from datastore import ndb
+from google_cloud_utils import big_query
+from handlers import base_handler
+from libs import handler
+
+# BigQuery query for calculating multi-armed bandit probabilities for
+# various strategies using a Boltzman Exploration (softmax) model.
+
+# Averages standardized new_edges feature over each strategy for expected
+# new_edges metric for each strategy.
+# See https://www.cs.mcgill.ca/~vkules/bandits.pdf for formula.
+
+MULTI_ARMED_BANDIT_PROB = """
+SELECT
+  /* Calculate bandit weights from calculated exponential values. */
+  strategy,
+  strategy_exp / exp_sum AS bandit_weight,
+  strategy_count
```

Comment:
I asked this in an older comment but want to call specific attention to it. What do we use this count for once we run the query, if anything? It does seem interesting when analyzing potential problems but for the weights themselves do we actually want it?

It might be fine to keep it here if it is meant as a metrics thing, but in that case we could write it back to bigquery like we do for fuzzer weights, and remove it from the datastore entity.